README.md

```diff
@@ -1,4 +1,4 @@
-# qtop.py [![Build Status](https://travis-ci.org/lookfwd/qtop.svg?branch=master)](https://travis-ci.org/lookfwd/qtop)
+# qtop.py [![Build Status](https://travis-ci.org/qtop/qtop.svg)](https://travis-ci.org/qtop/qtop)
```

Comment:
thanks for this PR. 

It has become a bit polluted, i.e. the commits by sfranky are kind of irrelevant;
can you resync with develop or, open a new PR to make it look pristine?